app/views/comparisons/baseload_per_pupil/_table.html.erb

```diff
@@ -1,58 +1,22 @@
-<div class='text-right mt-2'>
-  <%= download_link(report, table_name, index_params) %>
-</div>
+<%= component 'comparison_table', headers: @headers, report: report, table_name: table_name, index_params: index_params do |c| %>
+  <% @results.each do |result| %>
+    <% c.with_row do |r| %>
+      <% r.with_school school: result.school, advice_page: advice_page %>
```

Comment:
No biggie but would the advice_page key be applicable to the whole table rather than just a school? i.e. should it be passed as a parameter to the comparison_table component, rather than to every school? Might be a bit fiddly to make it available to sub-components though.

Reply:
The PageNavComponent does some passing of parameters through to slots if you need an example

Reply:
Yes, that's probably better. I've reworked.